alica_engine/src/engine/AlicaEngine.cpp

```diff
@@ -23,38 +23,41 @@
 #include "engine/planselector/PlanSelector.h"
 #include "engine/teammanager/TeamManager.h"
 #include <engine/syncmodule/SyncModule.h>
-#include <engine/syncmodule/SyncModule.h>
 
 #include <supplementary/AgentIDManager.h>
 
-namespace alica {
+namespace alica
+{
 /**
  * Abort execution with a message, called if initialization fails.
  * @param msg A string
  */
-void AlicaEngine::abort(const std::string& msg) {
+void AlicaEngine::abort(const std::string& msg)
+{
     std::cerr << "ABORT: " << msg << std::endl;
     exit(EXIT_FAILURE);
 }
 
 /**
  * The main class.
  */
-AlicaEngine::AlicaEngine(supplementary::AgentIDManager* idManager, const std::string& roleSetName, const std::string& masterPlanName, const std::string& roleSetDir, bool stepEngine)
-        : stepCalled(false)
-        , planBase(nullptr)
-        , planSelector(nullptr)
-        , communicator(nullptr)
-        , alicaClock(nullptr)
-        , sc(supplementary::SystemConfig::getInstance())
-        , terminating(false)
-        , expressionHandler(nullptr)
-        , log(nullptr)
-        , auth(nullptr)
-        , variableSyncModule(nullptr)
-        , pap(nullptr)
-        , stepEngine(stepEngine)
-        , agentIDManager(idManager) {
+AlicaEngine::AlicaEngine(supplementary::AgentIDManager* idManager, const std::string& roleSetName, const std::string& masterPlanName,
+                         const std::string& roleSetDir, bool stepEngine)
```

Comment:
Some crazy formatting issue

Reply:
Can you manually correct it if the formatter does not work properly.